client/cypress/integration/Navigating_spec.js

```diff
@@ -0,0 +1,64 @@
+describe("Navigate the App", () => {
+  it("creates a user sent to the DB and sends the user to the log in page", () => {
+    cy.viewport(375, 667);
+    cy.visit("http://localhost:3000/register");
+    cy.url().should("include", "/register");
+
+    cy.location().should((loc) => {
+      expect(loc.host).to.eq("localhost:3000");
+      expect(loc.href).to.eq("http://localhost:3000/register");
+    });
```

Comment:
Not sure if you should test that you are on the /register page if you don't have any action in between.
These tests would work on nearly every project (it would fail if you have a redirect on that page).